screens/Scan/ScanLayout.tsx

```diff
@@ -28,11 +28,9 @@ export const ScanLayout: React.FC = () => {
         hint={controller.statusOverlay?.hint}
         onCancel={controller.statusOverlay?.onButtonPress}
         onStayInProgress={controller.statusOverlay?.onStayInProgress}
-        isHintVisible={controller.isStayInProgress}
-        isBleErrorVisible={controller.isBleError}
+        isHintVisible={controller.isStayInProgress || controller.isBleError}
         onRetry={controller.statusOverlay?.onRetry}
         progress={controller.statusOverlay?.progress}
```

Comment:
Remove progress prop as it's not being used

Reply:
Removed